sale_order_customer_no_autofollow/__manifest__.py

```diff
@@ -0,0 +1,15 @@
+{
+    "name": "Sale order partner no autofollow",
+    "summary": "Sale order partner no autofollow",
```

Comment:
summary must contain some valuable summary. 
Eg "Do not add customer as follower in Sales Orders"

Reply:
Ok, I'll fix it